CHANGELOG

```diff
@@ -1,6 +1,10 @@
 deluge-webapi changelog
 =======================
 
+v0.4.3
```

Comment:
Why 0.4.3, whereas 0.4.0 was never released. Let it be 0.4.0.

Reply:
Will do.